Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM debian:stable
+
+
+WORKDIR /usr/src/app
+RUN apt update && \
+apt install -y build-essential cmake git libcap-dev curl ninja-build libssl-dev libboost-dev expat unbound libsodium-dev  libpgm-dev libzmq3-dev
+RUN apt install -y  pkg-config
+RUN apt install -y libboost-all-dev
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get install -y nodejs
+RUN  curl -L https://github.com/zeromq/cppzmq/archive/v4.3.0.tar.gz > v4.3.0.tar.gz
+RUN  tar zxf v4.3.0.tar.gz
+RUN  cd cppzmq-4.3.0 &&  cmake . &&  make -j$physicalCpuCount &&  make install
```

Comment:
$physicalCpuCount isn't defined yet (minor thing, just wanted to note it, so it's not forgot later)